wagtail_airtable/mixins.py

```diff
@@ -28,6 +28,25 @@ class AirtableMixin(models.Model):
 
     airtable_record_id = models.CharField(max_length=35, db_index=True, blank=True)
 
+    def _find_adjacent_models(self) -> dict:
```

Comment:
Yes there is a reason! 

Originally I was going to do exactly what you suggested (and your method still works, too). The EXTRA_SUPPORTED_MODELS setting was added to prevent configs from getting too large. It did add quite a bit of extra logic to the entire application which Im not too satisfied with to be honest. Unfortunately I got too far in this package before realizing there needs to be additional model support and didnt want to re-write how the config works.

But what I'll do is demonstrate your example in the README. I think most people will use it your way, it's only the extreme cases where 20+ models need to be supported (and that could result in 20+ lines of config which could get kind of gross.. though not the end of the world I suppose).